### DISCUSSION TITLE
Use outside of components?

### DISCUSSION THREAD

**Discussion by RebeccaStevens** [nuxt/image#950] [Q&A]
I'd like to be able to have a single image file that is then used to generate the `favicon.ico` as well as the images needed for the pwa settings (`pwa-192x192.png` and `pwa-512x512.png`).

Is this possible with nuxt image?

**Comment by DavidDeSloovere** [comment]
You should check [Nuxt PWA](https://pwa.nuxtjs.org/icon) or [Vite PWA](https://vite-pwa-org.netlify.app/assets-generator/) for generation.
